docs/src/installing.rst

```diff
@@ -85,10 +85,12 @@ local copy of Iris::
   conda env create --force --file=requirements/ci/iris.yml
   conda activate iris-dev
 
-The ``--force`` option is used when creating the environment, this is optional
-and will force the any existing ``iris-dev`` conda environment to be deleted
-first if present.  This is useful when rebuilding your environment due to a
-change in requirements.
+.. note::
+
+  The ``--force`` option is used when creating the environment, but first
+  removes any previously existing ``iris-dev`` environment of the same name
+  beforehand.  This is particularly useful when rebuilding your environment
```

Comment:
I feel like this use of "beforehand" is redundant given the use of "but first".

Reply:
snap!

Reply:
Nice one, thanks guys :+1: 

Done